query/src/main/java/org/infinispan/search/mapper/mapping/impl/InfinispanContainedTypeContext.java

```diff
@@ -31,5 +32,16 @@ public void identifierMapping(IdentifierMapping identifierMapping) {
       public void dirtyFilter(PojoPathFilter dirtyFilter) {
 
       }
+
+      @Override
+      public void dirtyContainingAssociationFilter(PojoPathFilter filter) {
+         PojoContainedTypeExtendedMappingCollector.super.dirtyContainingAssociationFilter(filter);
+      }
+
+      @Override
+      public void applyLoadingBinder(Object o,
+                                               PojoEntityLoadingBindingContext pojoEntityLoadingBindingContext) {
+
```

Comment:
there is nothing here

Reply:
I'm pretty sure you don't need loading for "contained" (non-indexed) entities in Infinispan. So leaving this empty should be safe.

And TBH I doubt this method is even ever called in Infinispan. You could make it throw an exception to be sure.